pyproject.toml

```diff
@@ -189,7 +190,7 @@ markers = [
 addopts = "--ignore=smoketests"
 
 [tool.ruff]
-src = ["truss", "truss-chains"]
+src = ["truss", "truss-chains", "truss-train"]
```

Comment:
This allows `truss-train` imports to be labeled as first-party for import sorting. Note for followup, I think we actually want to specify `.` instead of `truss` here, since right now `truss` is being categorized as third party across the [board](https://github.com/basetenlabs/truss/blob/536a1f7af1f04b7fd93814d7f69b566dd89aaac1/truss-train/truss_train/definitions.py?plain=1#L3-L5)